UNRELEASED.md

```diff
@@ -43,6 +43,11 @@
 
 ### Development workflow
 
+- Added `dev test:coverage` command ([#2496](https://github.com/Shopify/polaris-react/pull/2496))
+- Added `dev open coverage` command ([#2496](https://github.com/Shopify/polaris-react/pull/2496))
```

Comment:
```suggestion
- Added `dev open coverage` command ([#2496](https://github.com/Shopify/polaris-react/pull/2496)) to run `posttest:coverage`
```

Reply:
Is that what it does? I also wonder if it's not best to put the yarn command first in the changelog. Just because when I read the first 2 entries I wasn't sure what they would do until I read the yarn entries.

Reply:
`dev open coverage` runs `open build/coverage/index.html` so there is no yarn alias although I can add one.